Bug fixes.

calebkrauter opened the pull request:
Fixed a bug where crosshair and miner wouldn't load properly on Chrome, appeared to be caused by a different issue which I solved.

Fixed an issue where animations of Chad were getting cut out. I decided to shorten some of his animations for a quick fix. There may still be an off by one error but this does the trick.


Added merchant to "hill down from" main zone so that player can buy from merchant upon death or traveling back since everyone is scared and not at the village.

Game appears to be playable from start to finish in chrome. I intend to play test again after on "merge".

I added roast turkey to the eel boss and made it a bit faster and I made it so it would pursue as soon as boss music starts. 

One little issue is with the little eels which do not inflict damage and the music still is having trouble starting upon play. UPDATE: my browser did not allow auto playing of audio or video, after enabling that, it plays the music.

Commit:
Bug fixes.

Fixed a bug where crosshair and miner wouldn't load properly on Chrome, appeared to be caused by a different issue which I solved.

Fixed an issue where animations of Chad were getting cut out. I decided to shorten some of his animations for a quick fix. There may still be an off by one error but this does the trick.

Increased speed of eel boss and made it drop Roast Turkey.

Added merchant to "hill down from" main zone so that player can buy from merchant upon death or traveling back since everyone is scared and not at the village.

Game appears to be playable from start to finish in chrome. I intend to playtest again after on "merge".

NathanHinthorne commented on js/scripts/conversation_loaders/village_conversation_loader.js:
```diff
@@ -182,6 +182,51 @@ const blacksmithConversationLoader = () => {
             new DialogBubble(bs, "mmhm...", true),
             // 9
             new DialogBubble(bs, "Gonna need more than that.", true)
+        ], merchantScared: [
```
I like the addition of the merchant after the village attack. Now there's actually a reason to collect runes :)

calebkrauter replied:
Yup! Was the easiest fix to that problem I could think of. Now dying and respawning at village will actually be useful since you keep your items and runes upon death and can use your runes before going back to fight! It is helpful for the game-play loop similar to how in say Elden Ring when you want to grind and farm beasts for runes where you would just reset at the grace after defeating them you can just collect runes and upon death use them before fighting again. @tclaar at one point you offered to take on setting the respawn for Chad upon death to the current zone, I think we should not do that now for the reasoning I listed above.

NathanHinthorne reviewed (approved):
Good work @calebkrauter !

Commit:
Update

Added html info for user to autoplay audio for music and suggesting Chrome.

Added death upon falling too far in the end.

Commit:
Move wizard close to edge.
